### ISSUE TITLE
Rephrase neumes documentation on gapped va. non-gapped and connected vs unconnected neumes

### ISSUE THREAD

**Issue by bwbohl** [music-encoding/music-encoding#1251]
As surfaced from @ahankinson suggesting a minor change in:

>  <p>Broadly speaking, Western neume notations belong to two main categories. On one side we have notations where two or more notes were represented by a single pen-stroke, while on the other side there are notations where the notes are graphically separated by means of discrete dots or short pen-strokes. These distinctions have been described even within single notational styles as gapped or not gapped (Behrendt, Bain, &amp; Helsen 2017). To date, the MEI Neumes Module has been tested mainly on square notations and stroke notations (St. Gall, Old Hispanic, etc.), but also on Aquitanian point-notation.</p>

Originally posted by @ahankinson in https://github.com/music-encoding/music-encoding/pull/1230#discussion_r1234817669

**Comment by bwbohl** [comment]
@ahankinson, is this still an issue, or does the current dev version suffice?

**Comment by martha-thomae** [comment]
Summarizing the issue here, using @ahankinson's original comments:

> Regarding this paragraph:
>
>> Broadly speaking, Western early notations belong to two main categories. On one side we have notations where two or more notes were represented by a single pen-stroke, while on the other side there are notations where the notes are graphically separated by means of discrete dots or short pen-strokes. These distinctions have been described even within single notational styles as gapped or not gapped (Behrendt, Bain, & Helsen 2017). **To date, the MEI Neumes Module has been tested mainly on square notations and stroke notations (St. Gall, Old Hispanic, etc.), but also on Aquitanian point-notation.**
>
> The last sentence is a non-sequitur, for a couple reasons:
> 
>  1. The reader has no connection between the previous sentence about "gapped" vs. "ungapped" and this one so they're left uncertain about why it's being mentions. Are all of the notations mentioned gapped, or ungapped? Or does the list represent a set of neume notations that represent both gapped and ungapped styles?
>  2. I did a `cmd-f` search in the neumes chapter, and save for one caption this is the only mention of 'gapped' or 'ungapped' in the whole chapter; the one exception being an example of Old Hispanic notation, which has quite specialized connections anyway ("looped", "extended".). I would expect somewhere to mention the availability of `con="g"`, and that *no* `@con` attribute present means that they are connected (at least, that's what I understand from not having any explicit "ungapped" value on `@con`). So if this was fundamental to understanding Western neume notation and the design of the module, I would expect it to be followed-up later on with some further explanation of its impact on the specific encoding.

I think that the sentence can be separated from the rest of the paragraph to separate the ideas of "gapped / non-gapped" and "notations supported in MEI Neumes so far." Also, I agree with @ahankinson that a clarification of the use of `@con = g` (gapped) should be provided somewhere (indicating that the default would be ungapped and that one should indicate the gapped connection by using the attribute `@con` and value `g`).

**Comment by martha-thomae** [comment]
@fujinaga do you agree with this correction? And would there be someone willing to include this in the chapter, @ahankinson, @vigliensoni, maybe someone at DDMAL (@fujinaga)? We are accepting corrections to the guidelines now in the current milestone, before proceeding to the next one where we will move everything into PureODD (this will be happening by the end of November).

**Comment by fujinaga** [comment]
Yes, I agree with the correction. We don't really have much expertise in MEI encoding at this time at the DDMAL lab.

**Comment by martha-thomae** [comment]
Suggestion by Elsa De Luca to rewrite this as:

> Broadly speaking, Western early notations belong to two main categories. On one hand there are notations where two or more notes were represented by a single pen-stroke (stroke notations), while on the other hand there are notations where the notes are graphically separated by means of discrete dots or short pen-strokes (point notations). While this division is useful for categorization, sometimes these categories overlap and different kinds of early music scripts could express two (or more) notes with an uninterrupted pen-stroke. Furthermore, the use of a spatial gap to separate notes within the same neume is believed to have something to do with the vocal performance (articulation) of the relative notes (especially in stroke notations). The style of neumatic connection within a neume is represented in MEI with `@con`. To date, the MEI Neumes Module has been tested mainly on square notations and stroke notations (St. Gall, Old Hispanic, etc.), but also on Aquitanian point-notation.

I can make this change myself. If having enough time I will do it for this milestone (MEI 6.0-1+meiODDs). If not, I will include this change after moving into PureODD.
